api/MonitorizareVot.Ong/src/MonitorizareVot.Api/Services/IFileLoader.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using MonitorizareVot.Domain.Ong.ValueObjects;
+
+namespace MonitorizareVot.Ong.Api.Services
+{
+    public interface IFileLoader
+    {
+        Task<List<PollingStationDTO>> importFileAsync(IFormFile file);
```

Comment:
We should use Pascal Case for the function names in C#. If this was intentional, then we should make sure the documentation for the project includes this naming convention.

Reply:
No, it was not intentional, I will have a look at all the function names and convert them to Pascal Case.